### PR TITLE
Add ddev based development setup to readme

### PR DESCRIPTION
**Pull request by hhirsch**
### Description (*)
Add dev system setup with ddev

### Manual testing scenarios (*)
Try the instructions and a testing frontend/admin should be available

### Questions or comments
Please try and critique

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,13 @@ You can add additional meta files in this directory to cover your own project fi
 [PhpStorm advanced metadata](https://www.jetbrains.com/help/phpstorm/ide-advanced-metadata.html)
 for more information.
 
+## Development Environment with ddev
+- Install ddev
+- Clone the repository as described in Installation -> Using Git
+- Create a ddev config using ```$ ddev config``` the defaults should be good for you
+- Open .ddev/config.yaml and change the php version to 7.2
+- Navigate to http://magento-lts.ddev.site
```
Comment (kkrieger85) [inline README.md:83]: With DDEV you could navigate to https://...

File: README.md [hunk 2]
```diff
@@ -75,6 +75,13 @@ You can add additional meta files in this directory to cover your own project fi
 [PhpStorm advanced metadata](https://www.jetbrains.com/help/phpstorm/ide-advanced-metadata.html)
 for more information.
 
+## Development Environment with ddev
+- Install ddev
```
Comment (kkrieger85) [inline README.md:79]: Could you please add a Link to DDEV here?

File: README.md (hunk 1 above)
Comment (hhirsch) [reply README.md:83]: @kkrieger85 It isn't set up properly in my env. yet. Something went wrong with the mkcert.

File: README.md (hunk 2 above)
Comment (hhirsch) [reply README.md:79]: Yes.

File: README.md (hunk 1 above)
Comment (hhirsch) [reply README.md:83]: Solution fix it and change the link to https?

File: README.md (hunk 2 above)
Comment (hhirsch) [reply README.md]: Should be satisfactory resolved with the newest commit.

File: README.md (hunk 1 above)
Comment (hhirsch) [reply README.md]: Should be resolved in the newest commit.

**Review by kkrieger85 — APPROVED** [review]
LGTM

Thanks for your PR

**Review by Flyingmana — APPROVED** [review]

**Pull request merged**